alerts/utilization.libsonnet

```diff
@@ -31,7 +31,7 @@
             },
             annotations: {
               message: 'Storage cluster is critically full and needs immediate expansion',
-              description: 'The utilization of storage cluster has crossed 95%.',
+              description: 'Storage cluster utilization has crossed 95%.',
```

Comment:
we can add "Storage cluster expansion is required immediately.

Reply:
Same as above